aiohttp/web_exceptions.py

```diff
@@ -99,6 +99,8 @@ def __init__(
         super().__init__()
         if reason is None:
             reason = self.default_reason
+        elif "\n" in reason:
+            raise ValueError("Reason cannot contain \\n")
```

Comment:
```suggestion
            raise ValueError(r"Reason cannot contain \n")
```

Reply:
I originally felt the escape made it more explicit that this was deliberate, but can change it if you think it's more readable.